genai-perf/docs/multi_modal.md

```diff
@@ -45,23 +46,120 @@ docker run --runtime nvidia --gpus all \
     --model llava-hf/llava-v1.6-mistral-7b-hf --dtype float16
 ```
 
+Use GenAI-Perf to generate/send text and image request data to the server
+```bash
+genai-perf profile \
+    -m llava-hf/llava-v1.6-mistral-7b-hf \
+    --tokenizer llava-hf/llava-v1.6-mistral-7b-hf \
+    --service-kind openai \
+    --endpoint-type multimodal \
+    --image-width-mean 50 \
+    --image-height-mean 50 \
+    --synthetic-input-tokens-mean 10 \
+    --output-tokens-mean 10 \
+    --streaming
+```
 
-## Approach 1: Synthetic Multi-Modal Data Generation
+Console output will have the following result table
 
-GenAI-Perf can generate synthetic multi-modal data such as texts or images using
-the parameters provide by the user through CLI.
+```bash
+                           NVIDIA GenAI-Perf | Multi-Modal Metrics
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┓
+┃                         Statistic ┃    avg ┃    min ┃      max ┃    p99 ┃    p90 ┃    p75 ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━┩
```

Comment:
Do we want to show this table as an example if it does not provide non text metrics?

Reply:
okay. i wanted to make sure.